src/platform.rs

```diff
@@ -136,6 +136,7 @@ pub fn binary_path() -> anyhow::Result<PathBuf> {
     Ok(dir.join(BRANDING_CLI_CMD_FILE))
 }
 
+#[allow(dead_code)]
```

Comment:
Or, I guess this is so trivial we can delete it until we need it?

Reply:
sure!